pest_meta/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "pest_meta"
+version = "0.1.0"
+authors = ["Victor-Nicolae Savu <victor.nicolae.savu@gmail.com>"]
+repository = "https://github.com/pest-parser/pest"
+documentation = "https://docs.rs/pest"
+keywords = ["pest", "parser", "peg", "grammar"]
+categories = ["parsing"]
+license = "MPL-2.0"
+readme = "_README.md"
+
+[dependencies]
+maplit = "^0.1.5"
+pest_derive = "1.0.3"
+pest = "1.0.3"
+
+[dev_dependencies]
```

Comment:
This should be `dev-dependencies`. Not sure I'd keep this here though. It's kind of pointless to have every debug build download a copy of `colored`.

Reply:
Any suggestions as to what to do with the example code that depends on `colored`?